### ISSUE TITLE
[About the version with CTC Loss, and TTS application]

### ISSUE THREAD

**Issue by sh-lee-prml** [Stability-AI/stable-codec#7]
Hi @julian-parker  

Thanks for releasing the source code and checkpoints!

I noticed that the released models were trained with CTC Loss together during pre-training. 

[Q1. performance of the CTC version]

I would appreciate it if you could provide the objective evaluation results of CTC version compared to paper version. 

[Q2. multi-lingual capability of CTC models]

The original TAAE showed excellent results across various of languages. I am curious whether the model, trained with an English dataset using CTC Loss, still has robust multi-lingual capability on the unseen language? 

[Q3. TTS Application on the TAAE tokens]

Are there any plans to release the TTS models? If it is, could you share which codecs (400, 700, and 1000 BPS) are used? I have a plan to fine-tune the decoder of TAAE :)

Thank you once again for your nice work!

**Comment by julian-parker** [comment]
Hey @sh-lee-prml!

Thanks for your interest!

Q1: Yes, good point. We have those results. I'll add them to the README.
Q2: That's a good question. We haven't tested this extensively. I'll look into whether it's viable for us to get these results. If there's demand from the community it's also possible we could release the model checkpoint before the CTC finetune (so identical to results in the paper).
Q3: The TTS tests we've done are purely for internal use. There's no current plan to release them.

Hope that helps!

**Comment by sh-lee-prml** [comment]
Thank you so much 👍
